tests/test_membership.py

```diff
@@ -0,0 +1,171 @@
+# Copyright 2018 Comcast Cable Communications Management, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TODO: Add module docstring."""
+import datetime
+
+import responses
+
+from vinyldns.membership import Group, GroupChange, ListAdminsResponse, ListGroupsResponse, ListGroupChangesResponse, \
+    ListMembersResponse, Member, User
+from vinyldns.serdes import to_json_string, from_json_string
+from sampledata import sample_group, sample_group2
+
+
+def check_groups_are_same(a, b):
+    if a is None and b is None:
+        assert a == b
+    else:
+        assert a.id == b.id
+        assert a.description == b.description
+        assert a.created == b.created
+        assert a.name == b.name
+        assert all([l.__dict__ == r.__dict__ for l, r in zip(a.members, b.members)])
+        assert all([l.__dict__ == r.__dict__ for l, r in zip(a.admins, b.admins)])
+
+
```

Comment:
Important Note: I only tested the "happy" path in all of these unit tests.  We stage the response of the call.

You may ask "Paul: it doesn't seem like we are testing anything here".

Well Jimmy, actually if a staged response is not called, the test will fail so we are verifying the HTTP verb and URI are actually invoked.  Further, we are testing that the JSON is deserialized in the client to the appropriate model class (e.g. Group).

I covered all of the existing methods on the client with the unit tests, as well as some other serialization.